apps/server/src/shared/providers.test.ts

```diff
@@ -1,206 +0,0 @@
-import type { CachedInstitution } from "./contract";
```

Comment:
Why are we deleting this file?

Reply:
This file was copied and renamed to `aggregators.test.ts`, but the original was accidentally left. If you look at the `aggregators.test.ts` in the same folder, the contents is identical to this file I'm deleting.